enforcer-rules/src/main/java/org/sonatype/ossindex/maven/enforcer/BanVulnerableDependencies.java

```diff
@@ -312,7 +314,12 @@ private void maybeApplyProxy(final OssindexClientConfiguration clientConfigurati
         artifactFilter = new CumulativeScopeArtifactFilter(scopes);
       }
 
-      DependencyNode node = graphBuilder.buildDependencyGraph(project, artifactFilter);
+      ProjectBuildingRequest buildingRequest =
+              new DefaultProjectBuildingRequest( session.getProjectBuildingRequest() );
+      buildingRequest.setProject( project );
+
+      buildingRequest.setProject( project );
```

Comment:
duplicate call to setProject() ?

Reply:
Thanks @jdillon! You are right. I removed the duplicate call.